keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -1041,6 +1530,113 @@ representation. For admission control, we also need CEL to be integrated with
 the Kubernetes Go structs used to representative native API types, both for type
 checking and for runtime data access.
 
+### Writing to Status
+
+This enhancement proposes using status to of `ValidatingAdmissionPolicy` to
+communicate type-checking errors and any other misconfigurations such as CRD not
+found errors.
+
+As mentioned in
+https://github.com/kubernetes/enhancements/pull/3492#discussion_r964841045,
+status on API server configuration objects has been tricky to design in the
+past, because of the following:
+
+- multiple active kube-apiservers (sometimes at identical versions, sometimes
+  skewed by one version during upgrade)
+- multiple active non-kube-apiserver servers (aggregated servers)
+
+As a concrete example, The CRD `NonStructural` status field takes advantage of
+the metadata generation field
+(https://github.com/kubernetes/kubernetes/commit/2cfc3c69dc7c17b2711af0168f39ed7f515675c2).
+This avoids repeated updates for the same generation and potential fights of API
+server in HA environments even without leader elected controllers.
+
+We will use a similar approach. The complication here is that for this case we
+must consider up to 3 resources:
+
+- `ValidatingAdmissionPolicy` resource
+- Configuration CRD
+- If policy matches a CRD, the matched CRD
```

Comment:
"matches a CRD" == "matches a configuration resource"?

Reply:
Or is this the CRD that defines the kind-under-test, to be used for type checking?

Reply:
Yes, I'll rephrase.